utils/admin_console_plugin_index.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {stripMarkdown} from 'utils/markdown';
+
+function extractTextsFromPlugin(plugin) {
+    const texts = [];
+    if (plugin.name) {
+        texts.push(plugin.name);
+    }
+    if (plugin.id) {
+        texts.push(plugin.id);
+    }
+    if (plugin.settings_schema) {
+        if (plugin.settings_schema.footer) {
+            texts.push(plugin.settings_schema.footer);
+        }
+        if (plugin.settings_schema.header) {
+            texts.push(plugin.settings_schema.header);
+        }
+
+        if (plugin.settings_schema.settings) {
+            for (const setting of Object.values(plugin.settings_schema.settings)) {
+                if (setting.label) {
+                    texts.push(stripMarkdown(setting.label));
```

Comment:
For `label` and `display_name` I don't believe it's necessary to strip markdown given that it is not supported in these fields.